package.json

```diff
@@ -4,20 +4,19 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "npm run remove-dist && npm run build:production && npm run build:static",
-    "build:production": "webpack -p --mode=production",
-    "build:test": "webpack -p --mode=production --include-tests",
-    "build:dev": "npm run remove-dist && npm run build:static && webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
+    "build": "npm run build:production && npm run build:static",
+    "build:production": "webpack --mode=production",
+    "build:test": "build:production -- --include-tests",
```

Comment:
😮 You can do that and it works? Huh, I need to stop prefixing all of my scripts with `npm run` then 😆